ppr-ui/src/components/mhrTransfers/BusinessSearchAutocomplete.vue

```diff
@@ -0,0 +1,217 @@
+<template>
+  <v-card v-if="showAutoComplete" class="auto-complete-card" elevation="5">
```

Comment:
An id here might be helpful when there is capacity for unit tests.

Reply:
Added in next commit.